src/components/dropdown/Dropdown.vue

```diff
@@ -336,6 +342,18 @@ export default {
                     zIndex: '99'
                 }
             }
+        },
+
+        activeWindowScrollClose() {
```

Comment:
The `const func` should be added as a method called `onScroll` instead
Because the event listener needs to be cleaned in the `beforeDestroy` hook
(Or you may pollute the windows with event listeners if the dropdown is destroyed while it was opened)

Reply:
The problem I see with `onScroll` is that window can only be assigned one value? I'm assuming this is going to cause issues if anyone else is wanting to use `onScroll` or accidentally doesn't realize it's being used.

"Only one onscroll handler can be assigned to an object at a time." https://developer.mozilla.org/en-US/docs/Web/API/GlobalEventHandlers/onscroll

I'm trying to find a clean solution to remove the addEventListener, not sure yet.

Reply:
Also thanks for the help, appreciate it

Reply:
Sorry if my comment wasn't clear, I wasn't talking about the windows method, but the component methods and lifecycle hooks instead of having a const func, you need to reference `window.removeEventListener('scroll', this.onScroll)`
and `window.addEventListener('scroll', this.onScroll)`

And you will add a hook to the component
```
beforeDestroy() {
  window.removeEventListener('scroll', this.onScroll)
}
```
So that it cleans the event listener correctly